finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,43 @@
+var totalFrogs = 14;
+
+var numberWordsLowercase = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eightteen", "nineteen", "twenty"];
+var numberWordsCapitalized = ["Zero", "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Eleven", "Twelve", "Thirteen", "Fourteen", "Fifteen", "Sixteen", "Seventeen", "Eightteen", "Nineteen", "Twenty"];
```

Comment:
Nice job getting this challenge done. Note that this array is redundant. How could you use the javascript capitalize string method in this file?